src/sentry/utils/apidocs.py

```diff
@@ -246,8 +249,14 @@ def extract_endpoint_info(pattern, internal_endpoint):
             endpoint_name = endpoint_name[:-8]
         endpoint_name = camelcase_to_dashes(endpoint_name)
         title, text, warning, params = parse_doc_string(doc)
+        if not parents:
+            api_path = API_PREFIX + path.lstrip("/")
+        else:
+            parents_prefix = reduce(lambda x, y: "%s%s" % (x, y.lstrip("/")), parents)
```

Comment:
@manuzope wonder if we could do this more easily and more readily with a `map`+ `join`:
```
parents_prefix = "".join(map(lambda x: x.strip("/"), parents))
```